sources/ca/bc/nanaimo.json

```diff
@@ -5,17 +5,20 @@
         "city": "Nanaimo",
         "geometry": { "type": "Point", "coordinates": [-123.96, 49.17] }
     },
-    "data": "https://www.nanaimo.ca/open-data-catalogue/Download/Index?container=nanaimo&entitySetName=Addressing&downloadID=74",
-    "website": "https://www.nanaimo.ca/open-data-catalogue",
+    "data": "https://nanmap.nanaimo.ca/arcgis/rest/services/NanMap/Polygons/MapServer/15",
```

Comment:
The existing source was missing some streets, specifically `650 Terminal Ave`, which is present in this source. This source also appears to be continuously updated.